DEPS

```diff
@@ -107,7 +107,7 @@ hooks = [
     'action': [
       'python',
       '-c',
-      'import os; os.chdir("src"); os.chdir("electron"); os.system("npm install")',
+      'import os, subprocess; os.chdir(os.path.join("src", "electron")); subprocess.check_call(["python", "script/lib/npm.py", "install"]);',
```

Comment:
it's weird to invoke python to cd and then invoke python :/

Would [the --prefix option](https://docs.npmjs.com/misc/config#prefix) work? i.e. `npm install --prefix src/electron`

Reply:
Works on [Linux](https://circleci.com/gh/electron/electron/118669?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), but doesn't work on [Windows](https://windows-ci.electronjs.org/project/AppVeyor/electron-i8wjp/build/1.0.4736).